dspace-oai/src/main/java/org/dspace/xoai/app/XOAI.java

```diff
@@ -288,11 +289,20 @@ private int index(Iterator<Item> iterator) throws DSpaceSolrIndexerException {
                     log.error(ex.getMessage(), ex);
                 }
                 i++;
-                if (i % 100 == 0)
+                if (i % 1000 == 0) {
+                    System.out.println(i + " items prepared so far...");
+                }
+                if (i % 10000 == 0) {
```

Comment:
Should we make this intermediate commit point *configurable*?  I'm hoping committing every 10,000 is a sane default, but just wondering out loud if it's worth making this a configuration setting.

Reply:
This could be moved to a constant.  I am not sure it would justify a new command line param.